CHANGELOG-1.0.md

```diff
@@ -0,0 +1,11 @@
+# 1.0.x
+
+## Bug Fixes
+
+- API-599: fix const not supported by PHP 5.6
```

Comment:
Shouldn't we first tag the 1.0.2 before merging 1.0 into 2.0 ?

Reply:
we can merge before tag - it's not important no ?

Reply:
Yes you're right ^^